src/adaPets.js

```diff
@@ -9,13 +9,29 @@ const BASE_URL = 'http://localhost:3000/pets/';
 
 // Option functions.
 const listPets = () => {
+  axios.get('http://localhost:3000/pets/')
```

Comment:
You probably should have used `BASE_URL` here:

```suggestion
  axios.get(BASE_URL)
```